Serverless troubleshooting guide (#13163)
DataDog/documentation: content/en/serverless/installation/go.md

If your Lambda function is running in a VPC, follow these [instructions][8] to ensure that the extension can reach Datadog API endpoints.

## Troubleshooting

> If you have trouble collecting the monitoring data after following the instructions above, refer to the [serverless monitoring troubleshooting guide][9] for help.

Reviewer: ```suggestion
If you have trouble collecting monitoring data after following the instructions above, see the [serverless monitoring troubleshooting guide][9].
```
